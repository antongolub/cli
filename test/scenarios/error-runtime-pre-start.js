'use strict'
require('../..')(main)

async function main () {
  console.log('started')
  console.log('finished')
}

main.test.example = 'will explode'
