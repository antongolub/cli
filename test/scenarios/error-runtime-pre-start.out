scenarios/error-runtime-pre-start.js:
main.test.example = 'will explode'
                  ^

TypeError: Cannot set property 'example' of undefined
    at Object.<anonymous> (scenarios/error-runtime-pre-start.js:)
